Exploring_Pegasus/README.md

```diff
@@ -0,0 +1,57 @@
+# Exploring the Pegasus Topology
+
+D-Wave's Advantage quantum computer implements a chip topology called Pegasus.
+The programs in this folder provide a few different ways to explore the qubits
+and embeddings possible with Advantage.  For a full technical description of
+the Pegasus topology, see [our whitepaper](https://dwavesys.com/sites/default/files/14-1049A-A_The_D-Wave_Advantage_System_An_Overview_0.pdf).
+
+## Program Descriptions
+
+Here we describe each program in this directory. Note that although these
+programs are not running any problems on the QPU, they will require a token
+that has access to an Advantage solver.
```

Comment:
You could consider adding "the token is necessary because the Advantage solver is queried, even though no problems are run."